### DISCUSSION TITLE
How to override the debian multi-arch control information

### DISCUSSION THREAD

**Discussion by amgoyal** [goreleaser/nfpm#830] [Q&A]
I am trying to build a multi-arch debian package using nfpm, package file is simple very basic, and at the end I am adding following to override the control information as follows 

```
overrides:
  deb:
    control: |
      Multi-Arch: same
```
      
 but this addition of this section results in this error 

```
 yaml: unmarshal errors:
  line 149: field control not found in type nfpm.Overridables
```

What's the correct way to override the control information? or Is there any other way to build a multi-arch deb package using nfpm?

**Comment by caarlos0** [comment]
probably need to make control overridable on nfpm.

will open an issue

**Reply by caarlos0** [reply]
actually control and multi-arch field don't even exist in the nfpm config

**Reply by amgoyal** [reply]
Yeah, I added code for supporting it. I'll share the patch.
